static/css/marble.css

```diff
@@ -642,13 +637,21 @@ h6{
 }
 
 .researcher-user-image{
+    width: 100%;
```

Comment:
you forgot this for the `.student-user-image` class too

Reply:
Might have been testing for how it looks without it, but that's been added back in.